### PR TITLE
feat: upgraded docker plugin version

### PR DESCRIPTION
**Pull request by agatakrajewska**
The upgraded docker plugin supports scaccing openjdk hashes.

- [x] Tests written and linted [ℹ︎](https://github.com/snyk/general/wiki/Tests)
- [ ] Documentation written [ℹ︎](https://github.com/snyk/general/wiki/Documentation)
- [x] Commit history is tidy [ℹ︎](https://github.com/snyk/general/wiki/Git)
- [x] Potential release notes have been inspected

### What this does

Bumping docker plugin version. The updated package supports openjdk binaries scanning, returning hashes in the pluginResult.

### CODE REVIEW

File: test/fixtures/key-binaries-deployment.yaml [hunk 1]
```diff
@@ -1,23 +1,26 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: node-deployment
+  name: key-binaries-deployment
```
Comment (Shesekino) [inline test/fixtures/key-binaries-deployment.yaml]: just ```binaries-deployment``` I think

File: package.json [hunk 2]
```diff
@@ -39,7 +39,7 @@
     "needle": "^2.4.0",
```
Comment (Shesekino) [inline package.json]: typo in commit message

Comment (Shesekino) [reply package.json]: also please explain the change slightly more thoroughly in the title, keep in mind these are the release notes our users will see:
```
upgraded docker plugin version (43e8022)
```
against something like
```
support scanning for OpenJDK binaries
```

File: test/integration/kubernetes.test.ts [hunk 3]
```diff
@@ -135,16 +135,28 @@ tap.test('snyk-monitor sends correct data to kubernetes-upstream after adding an
...
-    pluginResult.hashes[0],
+    nodePluginResult.hashes[0],
     '1fd7117ccedc1c673381f71cc611131cb8a47852a882e3d731381f14355a3346',
     'SHA256 for node v12.16.1 found',
   );
+
+  const openjdkPluginResult = JSON.parse(depGraphResult.dependencyGraphResults.openjdk);
+  t.ok('hashes' in openjdkPluginResult, 'snyk-docker-plugin contains key-binary hashes');
+  t.equals(openjdkPluginResult.hashes.length, 2, 'two openjdk hashes found in node image');
+  const expectedHashes = ['a2fec3e093fa5b7b105efa2cc28a7ff5e168ba13ace0fff43a769db8797b173e',
```
Comment (Shesekino) [inline test/integration/kubernetes.test.ts]:
```
const expectedHashes = [
  'a2fec3e093fa5b7b105efa2cc28a7ff5e168ba13ace0fff43a769db8797b173e', 
  '00a90dcce9ca53be1630a21538590cfe15676f57bfe8cf55de0099ee80bbeec4',
];
```

File: package.json (hunk 2 above)
Comment (agatakrajewska) [reply package.json]: on it 👍

File: test/fixtures/key-binaries-deployment.yaml (hunk 1 above)
Comment (agatakrajewska) [reply test/fixtures/key-binaries-deployment.yaml]: yeah, that's simpler 😄

File: test/integration/kubernetes.test.ts [hunk 4]
```diff
@@ -110,15 +110,15 @@ tap.test('snyk-monitor sends data to kubernetes-upstream', async (t) => {
...
+  t.plan(9);
 
-  const deploymentName = 'node-deployment';
+  const deploymentName = 'binaries-deployment';
   const namespace = 'services';
   const clusterName = 'Default cluster';
   const deploymentType = WorkloadKind.Deployment;
 
-  await kubectl.applyK8sYaml('./test/fixtures/node-deployment.yaml');
+  await kubectl.applyK8sYaml('./test/fixtures/key-binaries-deployment.yaml');
```
Comment (Shesekino) [inline test/integration/kubernetes.test.ts]: key-binaries --> binaries

File: test/integration/kubernetes.test.ts [hunk 5]
```diff
@@ -110,15 +110,15 @@ tap.test('snyk-monitor sends data to kubernetes-upstream', async (t) => {
   t.same(busyboxPluginResult.plugin.packageManager, 'linux', 'linux is the default package manager for scratch containers');
 });
 
-tap.test('snyk-monitor sends correct data to kubernetes-upstream after adding another deployment', async (t) => {
-  t.plan(6);
+tap.test('snyk-monitor sends key binary hashes to kubernetes-upstream after adding another deployment', async (t) => {
```
Comment (Shesekino) [inline test/integration/kubernetes.test.ts]: key binary --> binary

File: package-lock.json [hunk 6]
```diff
@@ -2474,9 +2645,9 @@
       }
     },
     "minimist": {
-      "version": "1.2.5",
-      "resolved": "https://registry.npmjs.org/minimist/-/minimist-1.2.5.tgz",
-      "integrity": "sha512-FM9nNUYrRBAELZQT3xeZQ7fmMOBg6nWNmJKTcgsJeaLstP/UODVpGsr5OhXhhXg6f+qtJ8uiZ+PUxkDWcgIXLw=="
+      "version": "0.0.8",
```
Comment (Shesekino) [inline package-lock.json]: this I believe introduces a vulnerability

File: test/integration/kubernetes.test.ts (hunk 4 above)
Comment (agatakrajewska) [reply test/integration/kubernetes.test.ts]: oops, thank you

File: package-lock.json (hunk 6 above)
Comment (agatakrajewska) [reply package-lock.json]: that should be fixed now 😱

**Pull request merged**